src/custom/components/ProgressBar/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { ProgressBarWrap, ProgressContainer, Progress, Label, FlexWrap, HiddenRange, ProgressVal } from './styled'
+
+interface progressBarProps {
```

Comment:
Nitpick: can you make the interface name capitalized?

That's a (non written?) standard we use in the project

```suggestion
interface ProgressBarProps {
```